### DISCUSSION TITLE
28th open maintainers call

### DISCUSSION THREAD

**Discussion by mxinden** [libp2p/rust-libp2p#4406] [General]
Hi everyone,

The rust-libp2p maintainers will meet tomorrow (Tuesday) 09:00 UTC. See [libp2p public events calendar](https://calendar.google.com/calendar/embed?src=libp2p.io_0q9682i3te7eanhe9q7ae1c58g%40group.calendar.google.com).

https://meet.jit.si/rust-libp2p-open-maintainers-call

Everyone is welcome to join, listen along, propose topics and join the discussions.

Please post below in case you already want to add an agenda item or list it in [our notes document](https://hackmd.io/xoHKlFuER0OVj1DGDuFROw).

See you all tomorrow!

**Comment by mxinden** [comment]
Maybe @oblique and @zvolin you want to join. Would be great to learn more about your use-case of https://github.com/libp2p/rust-libp2p/pull/4015 (in case you can share more). As an aside, your work was very helpful in testing https://github.com/libp2p/rust-libp2p/pull/4348.

Maybe @StemCll you want to join to discuss https://github.com/libp2p/rust-libp2p/pull/4371.

Maybe @alindima you want to discuss next steps for https://github.com/libp2p/rust-libp2p/issues/4284.

Maybe @dgarus you would like to brain-storm a bit on the usability improvements you had in mind working on https://github.com/libp2p/rust-libp2p/pull/4275.

**Comment by dariusc93** [comment]
Not sure if I can make it but something that i would like to see eventually discussed is the memory usage of libp2p-quic and how high it can be in comparison to go-libp2p for the same number of connections (which was discussed in https://github.com/libp2p/rust-libp2p/pull/4325).

**Reply by mxinden** [reply]
Sorry for not tagging you here @dariusc93. I was operating under the assumption that this time does not work well with your timezone in general?! Happy to discuss QUIC memory concerns synchronous if you like. Want to suggest another time that works well for you?

**Comment by mxinden** [comment]
As always, notes of the call are in https://hackmd.io/xoHKlFuER0OVj1DGDuFROw.
